x-pack/plugins/siem/public/components/timeline/body/renderers/system/generic_file_details.test.tsx

```diff
@@ -67,7 +67,7 @@ describe('SystemGenericFileDetails', () => {
               packageSummary="[packageSummary-123]"
               packageVersion="[packageVersion-123]"
               processExecutable="[packageExecutable=123]"
-              processPid="[processPid-123]"
+              processPid={123}
```

Comment:
Should all these `processPid`'s be a `number[]`?

Reply:
Only asking as the types are declared as:

``` typescript
processPid: number | null | undefined;
processExecutable: string | null | undefined;	
```

and `processExecutable`'s value still has its `[]`'s:

``` typescript
processExecutable="[packageExecutable=123]"
```

Reply:
Executable will be a string name for sure. According to the docs:
https://www.elastic.co/guide/en/ecs/current/ecs-process.html

Pid is a `long` and executable is a `keyword` so I think this is correct